src/client/routes/Home/index.tsx

```diff
@@ -13,6 +13,7 @@ import { SummaryCard, InfoCard, ViewContainer, NoWalletCard, Amount } from '@com
 import { Text } from '@components/common';
 import { toBN, halfWidthCss } from '@src/utils';
 import { getConfig } from '@config';
+import React from 'react';
```

Comment:
same here, if you can put it in the first line! :)

Reply:
also, tsx files has react imported as default so I think is a wrong import

Reply:
sorted :)